File: python-beginner-workbook/module-03-control-flow/examples/example-02-simple-menu.py
# example-02-simple-menu.py
# Demonstrates a while loop for an interactive menu

running = True

while running:
    print("\n=== Task Manager Menu ===")
    print("1. Add task")
    print("2. View tasks")
    print("3. Complete task")
    print("4. Exit")
    
    choice = input("\nEnter your choice (1-4): ")
    
    if choice == "1":
        task_name = input("Enter task name: ")
        print(f"✓ Added task: {task_name}")
    elif choice == "2":
        print("Your tasks will appear here...")
    elif choice == "3":
        print("Marking task as complete...")
    elif choice == "4":
        running = False
        print("Goodbye! Thanks for using Task Manager.")
    else:
        print("❌ Invalid choice. Please enter 1, 2, 3, or 4.")

print("\nProgram ended.")

File: csharp-beginner-workbook/module-03-control-flow/examples/example-02-simple-menu.cs
// Example 2: Simple Menu Loop
// This demonstrates a while loop that creates an interactive menu

string choice = "";

while (choice != "3")
{
    Console.WriteLine("\n=== Task Manager ===");
    Console.WriteLine("1. View Tasks");
    Console.WriteLine("2. Add Task");
    Console.WriteLine("3. Quit");
    Console.Write("Enter your choice (1-3): ");
    
    choice = Console.ReadLine();
    
    if (choice == "1")
    {
        Console.WriteLine("\nYour current tasks:");
        Console.WriteLine("- Buy groceries");
        Console.WriteLine("- Finish report");
        Console.WriteLine("- Call client");
    }
    else if (choice == "2")
    {
        Console.Write("Enter task name: ");
        string task = Console.ReadLine();
        Console.WriteLine($"✓ Added task: {task}");
    }
    else if (choice == "3")
    {
        Console.WriteLine("\nGoodbye! Have a productive day!");
    }
    else
    {
        Console.WriteLine("\nInvalid choice. Please enter 1, 2, or 3.");
    }
}

